e2e-test/cypress/e2e/runner/completeAForm.feature

```diff
@@ -20,7 +20,7 @@ Feature: Complete a form
       | Do you have any evidence?           | No, I don't have evidence |
       | Additional Info                     | the additional info       |
     When I submit the form adapter
-    Then I see "Application complete"
+    Then I don't see "Application complete"
```

Comment:
Why are these tests updated? Do we not want the behaviour to be the same

Reply:
Not sure I understand. They test needs updating because the behaviour is different. 

I changed the APPLICATION COMPLETED template to remove the misleading heading. The test still checks for it.

Ideally we prevent the back-end of the Form Runner from sending the user to that page altogether, but deciphering how the Form Runner does that takes much longer.

Will split it in a different PR.